src/meta/src/barrier/mod.rs

```diff
@@ -194,7 +198,11 @@ pub struct GlobalBarrierManager {
 /// Controls the concurrent execution of commands.
 struct CheckpointControl {
     /// Save the state and message of barrier in order.
-    command_ctx_queue: VecDeque<EpochNode>,
+    command_ctx_queue: BTreeMap<u64, EpochNode>,
```

Comment:
Would you mind documenting somewhere what `u64` is for, or using an alias to explain it? (like `type PrevEpoch = u64`)